envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -159,7 +159,8 @@ def get_log_dir_path():
         return os.path.abspath(LOG_DIR_PATH)
 
     def test_server_online(self, island_client):
```

Comment:
Make sure this is ran first (check out why it's the first one now)(maybe autouse fixture)

Reply:
@mssalvatore By default pytests run in order. Enforcing extra order would be good, but it requires either significant work-arounds or third party package, which is not worth the trouble for what it is.